boa/integrations/jupyter/browser.py

```diff
@@ -2,28 +2,37 @@
 This module implements the BrowserSigner class, which is used to sign transactions
 in IPython/JupyterLab/Google Colab.
 """
-import importlib.util
 import json
 import logging
 from asyncio import get_running_loop, sleep
+from itertools import chain
 from multiprocessing.shared_memory import SharedMemory
 from os import urandom
-from typing import Any
+from typing import Any, Awaitable, Callable, Optional
 
 import nest_asyncio
 from IPython.display import Javascript, display
 
+from ...rpc import RPC, RPCError
 from .constants import (
     ADDRESS_TIMEOUT_MESSAGE,
     CALLBACK_TOKEN_BYTES,
     CALLBACK_TOKEN_TIMEOUT,
-    MEMORY_LENGTH,
     NUL,
     PLUGIN_NAME,
+    RPC_TIMEOUT_MESSAGE,
+    SHARED_MEMORY_LENGTH,
     TRANSACTION_TIMEOUT_MESSAGE,
 )
 from .utils import convert_frontend_dict, install_jupyter_javascript_triggers
 
+try:
+    from google.colab.output import eval_js as colab_eval_js
+except ImportError:
+    colab_eval_js: Optional[
```

Comment:
can we avoid the type signature here? i don't think it tells us anything very interesting

Reply:
Done